platform/viewer/src/components/Snackbar/Snackbar.css

```diff
@@ -0,0 +1,235 @@
+.sb-container {
+  position: fixed;
+  width: 420px;
```

Comment:
Just curious, why `420px`?

Reply:
Good question, It was just a random value, but looking at the material specifications, the default width is set to `344px`, I'll change to `344px`. Any thoughts?